docs/UserGuide.adoc

```diff
@@ -84,32 +87,245 @@ Examples:
+* `RATE` must be a positive real number.
+* Any expenses entered by the user during the trip would be automatically converted to SGD.
+****
+
+==== Modifying expenses
+
+The following sections [3.4.4.1 - 3.4.4.4] allows a user to set and modify expenses incurred during the trip.
+
+===== Adding a new expense: `addexpense`
+
+Allow the user to add a new expense.
+
+Format: `addexpense DAY detail/DETAIL [amountSGD/AMOUNTSGD] [amoungOTHR/AMOUNTOTHR]`
+
+****
+* Adds a new expense on specified `DAY`. The day refers to a specific day of the trip. The day *must be a positive integer* and be *lesser than the total number of days of the trip*.
+* At least one of the optional fields must be provided.
+* If the user enters the expense in other currency, the amount will be automatically converted to SGD using the *exchange rate* provided.
+****
+
+Examples:
+
+* `addexpense 1 detail/iPHONE11 amountSGD/1100` +
+Adds an expense incurred on the 1st day of the trip of an `iPHONE11` for $1100 SGD.
+
+* `addexpense 1 detail/iPHONE11 amountOTHR/800` +
+Adds an expense incurred on the 1st day of the trip of an `iPHONE11` for $800. This amount will be converted to SGD using *exchange rate* provided.
+
+
+===== Editing an expense: `editexpense`
+
+Allow the user to edit an expense. The expense must be already added to the expense tracker.
+
+Format: `editexpense INDEX [detail/DETAIL] [amountSGD/AMOUNTSGD] [amountOTHR/AMOUNTOTHR]`
+
+****
+* Edits the expense at the specified `INDEX`. The index refers to the index number shown in the displayed expense list. The index *must be a positive integer* 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* An *exchange rate* is required if the user is entering the amount in another currency.
+* If the user enters the expense in other currency, the amount will be automatically converted to SGD using the *exchange rate* provided.
+****
+
+Examples:
+
+* `editexpense 1 detail/MacbookPro 13 amountSGD/3000` +
+Edit the details and amount paid in SGD of the 1st expense to be `MacbookPro 13` and `$3000 SGD` respectively.
+
+* `editexpense 1 detail/AirPods amountOTHR/300` +
+Edit the details and amount paid in other currency of the 1st expense to be `AirPods` and `$300` respectively. The amount will be automatically converted to SGD using *exchange rate* provided.
+
+===== Deleting an expense: `deleteexpense`
+
+Allow the user to delete an expense. The expense must be already added to the expense tracker.
+
+Format: `deleteexpense INDEX`
+
+Examples:
+*deleteexpense 1 +
+Delete the 1st item in the expense tracker.
+
+
+===== Listing down expenses for a day: `list`
+
+Allow the user to see their expense for a specific day of the trip as a list.
```

Comment:
Grammar: "**Allows** the ...", "**his/her expenses** for ..."

Suggestion: "Shows a list of all expenses for a specified day of the trip."